src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -169,12 +169,14 @@ public override async Task<RazorDocumentRangeFormattingResponse> RazorRangeForma
                 Options = request.Options
             };
 
-            response.Edits = await _requestInvoker.ReinvokeRequestOnServerAsync<DocumentRangeFormattingParams, TextEdit[]>(
+            var edits = await _requestInvoker.ReinvokeRequestOnServerAsync<DocumentRangeFormattingParams, TextEdit[]>(
                 Methods.TextDocumentRangeFormattingName,
                 serverContentType,
                 formattingParams,
                 cancellationToken).ConfigureAwait(false);
 
+            response.Edits = edits ?? Array.Empty<TextEdit>();
```

Comment:
Did this so formatting doesn't completely break if one of the language servers returned null.